modules/api-support-tiers.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * rest_api/understanding-api-support-tiers.adoc
+
+[id="api-support-tiers_{context}"]
+= API support tiers
+
+All commercially supported APIs, components, and features are associated under one of the following support levels:
+
+[discrete]
+[id="api-support-tier-1_{context}"]
+== Support tier 1
+APIs and AOEs are stable within a major release for a minimum of 12 months or 3 minor releases, whichever is longer. The release that introduces a new or revised API or AOE, and the two following minor releases (n, n+1, n+2).
```

Comment:
AOE?

Reply:
application operating environment (AOE); I'll add the definition to the modules too